src/pages/_includes/graphql/customer-output-24.md

```diff
@@ -5,7 +5,7 @@ Attribute |  Data Type | Description
 `compare_list` | CompareList | The contents of the customer's comparison list
 `created_at` | String | Timestamp indicating when the account was created
 `custom_attributes` | [CustomerAddressAttribute](#customeraddress-attributes) | Deprecated. Use `custom_attributesV2` instead
```

Comment:
The link here goes to customer address attributes, it should link to customer attributes instead.

Reply:
actually the link goes nowhere since this doesn't not exist on the page which you currently are